core/storage/job/gae_models.py

```diff
@@ -94,17 +112,39 @@ def get_recent_jobs(cls, limit, recency_msec):
 
     @classmethod
     def get_all_unfinished_jobs(cls, limit):
+        """Gets limit number of unfinished jobs.
+
+        Args:
+            limit: int. A limit on the number of jobs to return.
+
+        Returns:
+            list(Jobs) or None. A list of at most limit number
+            of unfinished jobs.
+        """
         return cls.query().filter(
             JobModel.status_code.IN([STATUS_CODE_QUEUED, STATUS_CODE_STARTED])
         ).order(-cls.time_queued_msec).fetch(limit)
 
     @classmethod
     def get_unfinished_jobs(cls, job_type):
+        """Gets jobs that are unfinished.
+
+        Args:
+            job_type: str. The type of jobs that may be unfinished.
+
+        Returns:
+            list(Jobs) or None. A list of all jobs that belong to a job_type
```

Comment:
When you say 'Add full stop at end' do you just mean add a period at the end of the sentence? Sorry for the silly question just wanted to verify.